### DISCUSSION TITLE
How to end/delete Sketchybar?

### DISCUSSION THREAD

**Discussion by R0botBoi** [FelixKratz/SketchyBar#588] [Q&A]
I'm afraid I got myself into something I don't fully understand.
I realized sketchybar isn't for me so I tried to turn it off.
I tried the guidelines in "Uninstall", but to no avail. I also tried deleting brew as a whole but the bar still exists.
How can I get rid of the bar? Any help would be appreciated.

**Comment by rjaramillo21** [comment]
@R0botBoi 

brew uninstall sketchybar
brew untap FelixKratz/formulae

Ignore if the brew fails to uninstall... proceed with these other 2 commands and you are good to go. 

pkill "sketchybar"

osascript -e 'tell application "System Events" to set autohide menu bar of dock preferences to false'
